internal_packages/keybase/lib/keybase.coffee

```diff
@@ -0,0 +1,64 @@
+_ = require 'underscore'
+request = require 'request'
+
+class KeybaseAPI
+  constructor: ->
+    @baseUrl = "https://keybase.io"
+
+  getUser: (key, keyType, cb) =>
+    if not keyType in ['usernames', 'domain', 'twitter', 'github', 'reddit',
+                       'hackernews', 'coinbase', 'key_fingerprint']
+      console.error 'keyType must be a supported Keybase query type.'
```

Comment:
I really like this being here—it's always a good idea to add assertions to things like this. Maybe it should just `throw new Error("bla")`? If it's really a "this is not going to work" sort of thing.